src/strategies/razor-network-voting/index.ts

```diff
@@ -34,43 +97,10 @@ export async function strategy(
   //symbol: string,
   snapshot: string
 ) {
-  const params = {
-    delegators: {
-      __args: {
-        where: {
-          delegatorAddress_in: addresses
-        } // delegatorAddress
-      }, // Amount_Delegated
-      staker: {
-        totalSupply: true,
-        stake: true,
-        staker: true
-      },
-      delegatorAddress: true,
-      sAmount: true
-    },
-    stakers: {
-      __args: {
-        where: {
-          staker_in: addresses //  stakerAddress
-        }
-      },
-      stake: true,
-      totalSupply: true,
-      staker: true,
-      sAmount: true
-    }
-  };
-
-  if (snapshot !== 'latest') {
-    // @ts-ignore
-    params.delegators.__args.block = { number: snapshot };
-  }
-
   const score = {};
 
-  // subgraph request 1 : it fetches all the details of the stakers and delegators.
-  const result = await subgraphRequest(RAZOR_NETWORK_SUBGRAPH_URL, params);
+  // subgraph request : it fetches all the details of the stakers and delegators.
+  const result = await getAllData(snapshot);
```

Comment:
How many network requests it can send? 😅

Reply:
Ideally, it should not send more than 5 requests https://github.com/snapshot-labs/snapshot-strategies#checklist-for-adding-a-new-voting-strategy-or-validation-strategy

Reply:
It's highly unlikely that we will surpass the upper limit of more than five requests.

Reply:
Then it should be cool :)